web_src/css/index.css

```diff
@@ -31,6 +31,7 @@
 @import "./form.css";
 @import "./repository.css";
 @import "./repository-release-tag.css";
+@import "./repository-issue-label.css";
```

Comment:
Move thse two files to `repository/issue-label.css` and `repository/release-tag.css`. Less files in CSS root is easier to manage.

Reply:
Next PR

Reply:
Also handle `editor-markdown.css` there.

Reply:
Would still prefer it in this PR.

Reply:
2d81c67620078e32ae40ff4425106d5b788547b3